src/applications/mhv/secure-messaging/tests/e2e/secure-messaging-inbox-user-navigation.cypress.spec.js

```diff
@@ -37,6 +37,6 @@ describe('Secure Messaging Compose', () => {
       .type('Test message body', { force: true });
     composePage.pushSendMessageWithKeyboardPress();
     composePage.verifySendMessageConfirmationMessage();
-    landingPage.verifyInboxHeader('Inbox');
+    // landingPage.verifyInboxHeader('Inbox');
```

Comment:
@fazilqa @haniasma looks like `verifyInboxHeader` function does not exist. Commenting out temporarily but bringing up to your attention